lib/versioning/node/index.spec.ts

```diff
@@ -10,22 +10,35 @@ describe('versioning/node/index', () => {
     DateTime.local = dtLocal;
   });
 
-  test.each`
-    currentValue | rangeStrategy | currentVersion | newVersion  | expected
-    ${'1.0.0'}   | ${'replace'}  | ${'1.0.0'}     | ${'v1.1.0'} | ${'1.1.0'}
-    ${'~8.0.0'}  | ${'replace'}  | ${'8.0.2'}     | ${'v8.2.0'} | ${'~8.2.0'}
-  `(
-    'getNewValue($currentValue, $rangeStrategy, $currentVersion, $newVersion, $expected) === $expected',
-    ({ currentValue, rangeStrategy, currentVersion, newVersion, expected }) => {
-      const res = nodever.getNewValue({
+  describe('getNewValue', () => {
+    test.each`
+      currentValue | rangeStrategy | currentVersion | newVersion   | expected
+      ${'1.0.0'}   | ${'replace'}  | ${'1.0.0'}     | ${'v1.1.0'}  | ${'1.1.0'}
+      ${'~8.0.0'}  | ${'replace'}  | ${'8.0.2'}     | ${'v8.2.0'}  | ${'~8.2.0'}
+      ${'erbium'}  | ${'replace'}  | ${'12.0.0'}    | ${'v14.1.4'} | ${'fermium'}
+      ${'Fermium'} | ${'replace'}  | ${'14.0.0'}    | ${'v16.1.6'} | ${'gallium'}
```

Comment:
I think, most users would like to see preserved capitalization, i.e. `fermium -> gallium` and `Fermium -> Gallium`

```suggestion
      ${'Fermium'} | ${'replace'}  | ${'14.0.0'}    | ${'v16.1.6'} | ${'Gallium'}
```

Reply:
Though, we may want to implement this in separate PR across all similar versioning schemes

Reply:
Yes, good idea to have a separate PR for preserving capitalization! Let's track this in https://github.com/renovatebot/renovate/issues/13585.